src/lambda.rs

```diff
@@ -0,0 +1,69 @@
+use aws_sdk_lambda::error::InvokeAsyncError;
+use bytes::Bytes;
+
+use aws_sdk_lambda::model::LogType::Tail;
+use aws_sdk_lambda::output::InvokeAsyncOutput;
+use aws_sdk_lambda::types::{Blob, SdkError};
+use aws_sdk_lambda::RetryConfig;
+use aws_smithy_types::base64;
+use aws_types::SdkConfig;
+
+pub struct LambdaClient {
+    client: aws_sdk_lambda::Client,
+}
+
+pub struct LambdaResult {
+    base64_logs: String,
+    payload: Vec<u8>,
+}
+
+impl LambdaResult {
+    pub fn get_logs(&self) -> String {
+        std::str::from_utf8(&base64::decode(&self.base64_logs).unwrap())
+            .unwrap()
+            .to_string()
+    }
+    pub fn get_payload(&self) -> String {
+        std::str::from_utf8(&self.payload).unwrap().to_string()
+    }
+}
+
+impl LambdaClient {
+    pub fn new(config: &SdkConfig) -> LambdaClient {
+        let lambda_config = aws_sdk_lambda::config::Builder::from(config)
+            .retry_config(RetryConfig::new().with_max_attempts(2))
+            .build();
+        let client = aws_sdk_lambda::Client::from_conf(lambda_config);
+        LambdaClient { client }
+    }
+
+    pub async fn invoke(&self, function_name: &str, payload: String) -> LambdaResult {
+        let res = self
+            .client
+            .invoke()
+            .function_name(function_name)
+            .payload(Blob::new(payload))
+            .log_type(Tail)
+            .send()
+            .await
+            .unwrap();
+        LambdaResult {
+            base64_logs: res.log_result.unwrap_or_else(|| "".to_string()),
```

Comment:
I think you don't really need `unwrap_or_else` and a closure here, `unwrap_or` should be enough:
```suggestion
            base64_logs: res.log_result.unwrap_or("".to_string()),
```